### DISCUSSION TITLE
Backspace to reset mods is nerfed

### DISCUSSION THREAD

**Discussion by 2minwia** [ppy/osu#25837] [Q&A] [answered]
Before todays update (2023.1218.0), one could hit backspace in the mods select menu to disable all of them. Which was really handy.
However, the search box being auto-selected make this not work, unless one first unselects it, which feels like it defeats the point of it being a shortcut, as at that point, one could just click the "deselect all mods" button instead (which is a lot slower than just hitting backspace)...
I would really apprecaite having the original functionality back! Or something like, if the textbox is empty, backspace still disables all mods?

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/25842
